lib/vauth/digest_sspi.c

```diff
@@ -666,6 +666,8 @@ void Curl_auth_digest_cleanup(struct digestdata *digest)
 
   /* Free the copy of user/passwd used to make the identity for http_context */
   Curl_safefree(digest->user);
+  if(digest->passwd)
+    Curl_explicit_bzero(digest->passwd, strlen(digest->passwd);
```

Comment:
```suggestion
    Curl_explicit_bzero(digest->passwd, strlen(digest->passwd));
```
This should fix AppVeyor CI builds.